packages/http-logger/package.json

```diff
@@ -1,32 +1,33 @@
-{
-  "name": "@sliit-foss/http-logger",
-  "version": "1.3.1",
-  "description": "Http logging middleware for Express.js",
-  "main": "dist/index.js",
-  "scripts": {
-    "build": "node ../../scripts/esbuild.config.js",
-    "build:watch": "bash ../../scripts/esbuild.watch.sh",
-    "bump-version": "bash ../../scripts/bump-version.sh --name=@sliit-foss/http-logger",
-    "lint": "bash ../../scripts/lint.sh",
-    "release": "bash ../../scripts/release.sh",
-    "test": "bash ../../scripts/test/test.sh"
-  },
-  "dependencies": {
-    "@sliit-foss/module-logger": "1.3.1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/sliit-foss/npm-catalogue.git"
-  },
-  "homepage": "https://github.com/sliit-foss/npm-catalogue/blob/main/packages/http-logger/readme.md",
-  "keywords": [
-    "logging",
-    "http-logging",
-    "middleware"
-  ],
-  "author": "SLIIT FOSS",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/sliit-foss/npm-catalogue/issues"
-  }
-}
+{
+  "name": "@sliit-foss/http-logger",
+  "version": "1.3.1",
+  "description": "Http logging middleware for Express.js",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "node ../../scripts/esbuild.config.js",
+    "types": "types/index.d.ts",
```

Comment:
This is in the wrong place, shall we bring it out ?

Reply:
Sorry it was mistakenly added. Fixed it.

Reply:
Added the requested changes.